src/sdk/tests/index.tsx

```diff
@@ -12,9 +12,8 @@ let renders = 0
 function TestProvider({ children }: PropsWithChildren<unknown>) {
   const [id, setId] = useState('')
 
-  renders++
-
   useEffect(() => {
+    renders++
```

Comment:
Looks like changes from https://github.com/vtex-sites/base.store/pull/312/files slipped through this PR.